automation_tools/__init__.py

```diff
@@ -428,8 +433,12 @@ def setup_abrt():
         '/etc/foreman-proxy/settings.d/abrt.yml')
 
     # run the required commands
-    run('systemctl start abrtd')
-    run('systemctl start abrt-ccpp')
+    if os_version >= 7:
+        run('systemctl start abrtd')
+        run('systemctl start abrt-ccpp')
+    else:
+        run('service abrtd start')
+        run('service abrt-ccpp start')
 
     # edit the config files
     run('sed -i -e "s|^URL = .*|URL = https://{0}:8443/abrt/|" '
```

Comment:
If you will test the task be aware that the file sed edits is not in place for RHEL 6. I still investigating/researching to find why.